.travis.yml

```diff
@@ -30,6 +30,8 @@ before_cache:
 
 matrix:
   include:
+    - compiler: "ghc-8.8.1"
```

Comment:
Instead of manually editing `.travis.yml`, I'd prefer to have this autogenerated with the [`haskell-ci`](https://github.com/haskell-CI/haskell-ci) tool. It should suffice to do the following:

1. Build `haskell-ci` from source at [its GitHub repo](https://github.com/haskell-CI/haskell-ci)
2. Add `8.8.*` to the `tested-with` stanza in `unix.cabal`
3. Run `haskell-ci regenerate` in the same directory as `unix.cabal`